### PR TITLE
DOCS-752: Make titles shorter and add an additional link

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/internals/robot-to-robot-comms.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Robot-to-Robot Communication: Application End-to-End Flow"
+title: "Robot-to-Robot Communication: Application E2E Flow"
```
Comment (npentrel) [inline docs/internals/robot-to-robot-comms.md:2]:
```suggestion
title: "Robot-to-Robot Communication: Application End-to-End Flow"
```

File: docs/tutorials/projects/send-security-photo.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Build a Person Detection Security Robot That Sends You a Photo of the Person Stealing Your Chocolates"
+title: "Build a Person Detection Security Robot That Sends You Photo"
```
Comment (andf-viam) [inline docs/tutorials/projects/send-security-photo.md:2]:
```suggestion
title: "Build a Person Detection Security Robot That Sends You Photos"
```

File: layouts/partials/head.html [hunk 3]
```diff
@@ -16,8 +16,8 @@
 <meta name="robots" content="noindex, nofollow" />
 {{ end -}} {{ partialCached "favicons.html" . }}
 <title>
-    {{- if .IsHome -}} {{ .Site.Title -}} {{ else -}} {{ with .Title }}{{ . }} |
-    {{ end -}} {{ .Site.Title -}} {{ end -}}
+    {{- if .IsHome -}} {{ .Site.Title -}} {{ else -}} {{ with .Title }}{{ . }}
+    {{ end -}} {{ end -}}
```
Comment (andf-viam) [inline layouts/partials/head.html:20]: Unsure: without `{{ .Site.Title -}}` do we still need both instances of `{{ end -}}`?

Comment (npentrel) [reply layouts/partials/head.html:20]: Yep - they're for the if and the with clause as far as I understand

**Pull request merged**